WagerMate/Components/Pages/Bet-overview.razor

```diff
@@ -319,7 +329,7 @@
             FilterBets();
         }
     }
-    private bool _displayClosedBets { get; set; } = false;
+    private bool _displayClosedBets { get; set; } = true;
```

Comment:
Why true instead of false now?

Reply:
We decided to display all bet-states when initially loading